ci/infra/azure/README.md

```diff
@@ -0,0 +1,160 @@
+# Introduction
+
+This terraform project creates the infrastructure needed to run a
+cluster on top of Azure.
+
+Once the infrastructure is up and running nothing special has to be done
+to deploy CaaS Platform on top of it.
+
+This document focuses on the key aspects of the infrastructure created
+by terraform.
+
+# Cluster layout
+
+## Machines
+
+As usual the cluster is based on two types of nodes: master and worker nodes.
+
+All the nodes are created using the SLES 15 SP1 container host image built
+and maintained by the SUSE Public Cloud team.
+
+Right now users **must** bring their own license into the public cloud to be
+able to access SLE and SUSE CaaS Platform packages.
+
+The machines are automatically registered at boot time against SUSE Customer
+Center, RMT or a SUSE Manager instance depending on which one of the following
+variables has been set:
+
+  * `caasp_registry_code`
+  * `rmt_server_name`
+  * `suma_server_name`
+
+The SLES images [do not yet support cloud-init](https://docs.microsoft.com/en-us/azure/virtual-machines/linux/using-cloud-init)
+(it will probably be supported starting from SLE15 SP2). In the meantime
```

Comment:
This needs to adjust. We will start with SLE15 SP2.

Reply:
Since cloud-init implement differently from vm-extension, we will need to update the Terraform script after SP2 available on Azure

Reply:
Going with the current implementation is fine from my point of view, we can update later on.